crashlogger.js

```diff
@@ -23,9 +23,9 @@ let hadException = false;
  * Logs when a crash happens to console, then e-mails those who are configured
  * to receive them.
  *
- * @param {Error} err
+ * @param {NodeJS.ErrnoException} err
```

Comment:
...never mind, now that I think about it, since `NodeJS.ErrnoException` extends from `Error` it'd be fine to use in functions taking `Error` as parametres if they only access its stack property, like here. I'll change it back